cmd/process-agent/main.go

```diff
@@ -24,6 +62,8 @@ func main() {
 	flag.StringVar(&opts.check, "check", "", "Run a specific check and print the results. Choose from: process, connections, realtime")
 	flag.Parse()
 
+	opts.isConfig = flag.Arg(0) == "config"
```

Comment:
Is this the same config arg as the one above?

https://github.com/DataDog/datadog-agent/pull/8693/files#diff-560e015f438c5a25f09d4ff0b122fe051018a2c749b80e1dabc8504377fccf82R52